retriever/__main__.py

```diff
@@ -138,15 +138,36 @@ def main():
                 from retriever import lscolumns
                 lscolumns.printls(dataset_names())
             else:
-                all_scripts = datasets(args.l[0])
-                print("Available datasets : {}\n".format(len(all_scripts)))
-                count = 1
-                for script in all_scripts:
-                    print("{}. {}".format(count, script.title))
-                    print(script.name)
-                    print(script.keywords)
-                    print()
-                    count += 1
+                if len(args.l) > 0:
```

Comment:
when using the CLI 
`retriever ls -l MIT GPL` let us search keywords and licence.
Than means in get_opts module we should add a list of licensee as we do for the key words.
- This will enable retriever ls -l [tab] to give you both keywords and licenses as possible options